index.bs

```diff
@@ -5520,34 +5522,18 @@ Each stored [=virtual authenticator=] has the following properties:
 : |hasUserVerification|
 :: If set to [TRUE], the authenticator supports [=user verification=].
 : |isUserPresent|
-:: Determines the result of a [=test of user presence=] performed on the [=Virtual Authenticator=]. If set to `true`,
-    a [=test of user presence=] will always succeed. If set to `false`, it will fail.
+:: Determines the result of a [=test of user presence=] performed on the [=Virtual Authenticator=]. If set to [TRUE],
+    a [=test of user presence=] will always succeed. If set to [FALSE], it will fail.
 : |isUserVerified|
-:: Determines the result of [=User Verification=] performed on the [=Virtual Authenticator=]. If set to `true`,
-    [=User Verification=] will always succeed. If set to `false`, it will fail. This property has no effect if
-    |hasUserVerification| is set to `false`.
+:: Determines the result of [=User Verification=] performed on the [=Virtual Authenticator=]. If set to [TRUE],
+    [=User Verification=] will always succeed. If set to [FALSE], it will fail. This property has no effect if
+    |hasUserVerification| is set to [FALSE].
 
 ## <dfn>Add Virtual Authenticator</dfn> ## {#sctn-automation-add-virtual-authenticator}
 
 The [=Add Virtual Authenticator=] [=extension command=] creates a software [=Virtual Authenticator=]. The
 WebDriver [=extension command=] is defined as follows:
```

Comment:
Upon reflection, I'd trim this down like so:
The [=Add Virtual Authenticator=] WebDriver [=extension command=] creates a software [=Virtual Authenticator=]. It is defined as follows:

Reply:
Done everywhere.